### PR TITLE
First pass at update the contribution instructions

### PR DESCRIPTION
**Pull request by tmeasday**
See https://github.com/storybooks/storybook/issues/756#issuecomment-297923933

We should actually ensure this works consistently for people before making this change.

### CODE REVIEW

**Review by shilman — APPROVED** [review]
Tested and these instructions look good to me.

However I am not sure about the difference between `test-cra` and `cra-storybook`. Storybook works in both directories, but `npm start` only works for me in `cra-storybook`. @ndelangen any input here?

**Comment by shilman** [comment]
My error for `npm start` in `test-cra`:

```
Failed to compile.

Error in   TypeError: Cannot read property '0' of undefined
  
  - ExternalModuleFactoryPlugin.js:19 
    [test-cra]/[webpack]/lib/ExternalModuleFactoryPlugin.js:19:40
  
  - NormalModuleFactory.js:159 
    [test-cra]/[react-scripts]/[webpack]/lib/NormalModuleFactory.js:159:3
  
  - Tapable.js:75 NormalModuleFactory.applyPluginsAsyncWaterfall
    [test-cra]/[react-scripts]/[tapable]/lib/Tapable.js:75:69
  
  - NormalModuleFactory.js:144 NormalModuleFactory.create
    [test-cra]/[react-scripts]/[webpack]/lib/NormalModuleFactory.js:144:8
  
  - Compilation.js:356 Compilation.process [as _addModuleChain]
    [test-cra]/[react-scripts]/[webpack]/lib/Compilation.js:356:16
  
  - Compilation.js:424 Compilation.process [as addEntry]
    [test-cra]/[react-scripts]/[webpack]/lib/Compilation.js:424:7
  
  - SingleEntryPlugin.js:24 Compiler.compiler.plugin
    [test-cra]/[webpack]/lib/SingleEntryPlugin.js:24:16
  
  - Tapable.js:107 Compiler.applyPluginsParallel
    [test-cra]/[react-scripts]/[tapable]/lib/Tapable.js:107:14
  
  - Compiler.js:394 Compiler.compile
    [test-cra]/[react-scripts]/[webpack]/lib/Compiler.js:394:7
  
  - Compiler.js:203 Compiler.runAsChild
    [test-cra]/[react-scripts]/[webpack]/lib/Compiler.js:203:7
  
  - compiler.js:70 
    [test-cra]/[html-webpack-plugin]/lib/compiler.js:70:19
  
  - compiler.js:69 Object.compileTemplate
    [test-cra]/[html-webpack-plugin]/lib/compiler.js:69:10
  
  - index.js:47 Compiler.<anonymous>
    [test-cra]/[html-webpack-plugin]/index.js:47:40
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,28 +15,55 @@ No software is bug free. So, if you got an issue, follow these steps:
...
+### Testing against `master`
+
+To test your project against the current latest version of storybook, you can clone the repository and link it with `npm`. Try following these steps:
+
+1. Download the latest version of this project, and build it
+
+  ```
+  git clone https://github.com/storybooks/storybook.git
+  cd storybook
+  npm install
```
Comment (UsulPro) [inline CONTRIBUTING.md]: At this moment when I `npm install` it only installs root dependencies. In order to install all packages, I need to run `npm run bootstrap`.
OS: Windows. So I can't tell is it common or only windows issue (but previosly `npm install` started to install everything inside packages folder)

Comment (tmeasday) [inline CONTRIBUTING.md]: @ndelangen where are we at with fixing this problem? Should we update these instructions or will it be sorted soon?

Comment (ndelangen) [inline CONTRIBUTING.md]: This should be changed so `npm run bootstrap` is a required step.

Comment (ndelangen) [inline CONTRIBUTING.md]: I added it

**Review by ndelangen — APPROVED** [review]

**Pull request merged**